From 52c847eeda80cdb95c0ea8aaf2cb8f6656b4aa5b Mon Sep 17 00:00:00 2001
From: trickypr <23250792+trickypr@users.noreply.github.com>
Date: Fri, 1 Dec 2023 17:04:29 +1100
Subject: [PATCH] 004 - Correct types for E10SUtils (predictOriginAttributes)

---
 toolkit/modules/E10SUtils.sys.mjs | 6 ++++++
 1 file changed, 6 insertions(+)

diff --git a/toolkit/modules/E10SUtils.sys.mjs b/toolkit/modules/E10SUtils.sys.mjs
index 2be1ab1394..6e415ffa00 100644
--- a/toolkit/modules/E10SUtils.sys.mjs
+++ b/toolkit/modules/E10SUtils.sys.mjs
@@ -808,6 +808,12 @@ export var E10SUtils = {
 
   /**
    * Assemble or predict originAttributes from available arguments.
+   * @param {object} options
+   * @param {Window} [options.window]
+   * @param {*} [options.browser]
+   * @param {number} [options.userContextId]
+   * @param {string} [options.geckoViewSessionContextId]
+   * @param {number} [options.privateBrowsingId]
    */
   predictOriginAttributes({
     window,
-- 
2.42.0

