From a1eae8dc5f19efa6fc6d9343373f838bb3c7ee15 Mon Sep 17 00:00:00 2001
From: trickypr <23250792+trickypr@users.noreply.github.com>
Date: Fri, 1 Dec 2023 17:07:59 +1100
Subject: [PATCH] 005 - Fix NetUtils type (newURI)

---
 netwerk/base/NetUtil.sys.mjs | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/netwerk/base/NetUtil.sys.mjs b/netwerk/base/NetUtil.sys.mjs
index 679c9979a7..06882dd4c6 100644
--- a/netwerk/base/NetUtil.sys.mjs
+++ b/netwerk/base/NetUtil.sys.mjs
@@ -158,12 +158,12 @@ export var NetUtil = {
    * Constructs a new URI for the given spec, character set, and base URI, or
    * an nsIFile.
    *
-   * @param aTarget
+   * @param {string} aTarget
    *        The string spec for the desired URI or an nsIFile.
-   * @param aOriginCharset [optional]
+   * @param [aOriginCharset]
    *        The character set for the URI.  Only used if aTarget is not an
    *        nsIFile.
-   * @param aBaseURI [optional]
+   * @param [aBaseURI]
    *        The base URI for the spec.  Only used if aTarget is not an
    *        nsIFile.
    *
-- 
2.42.0

